includes/class-algolia-cli.php

```diff
@@ -120,7 +125,7 @@ private function do_reindex( Algolia_Index $index, $clear ) {
 			WP_CLI::success( sprintf( __( 'Correctly cleared index "%s".', 'wp-search-with-algolia' ), $index->get_name() ) );
 		}
 
-		$total_pages = $index->get_re_index_max_num_pages();
+		$total_pages = $index->get_re_index_max_num_pages() - ( $from_page - 1 );
```

Comment:
Need to update `$from_page` to `$from_batch`

Reply:
Done. Well spotted!